backend-shared/config/db/util/utils.go

```diff
@@ -467,24 +509,26 @@ func GetOrCreateDeploymentToApplicationMapping(ctx context.Context, createDeplTo
 		return false, nil
 	}
 
+	log = log.WithValues(createDeplToAppMapping.GetAsLogKeyValues()...)
+
 	// At this point in the function, the database row necessarily does not exist.
 
 	// Ensure that there isn't an old depltoappmapping hanging around with matching name/namespace, before we create a new one.
-	if _, err := dbq.DeleteDeploymentToApplicationMappingByNamespaceAndName(ctx,
+	if resultsDeleted, err := dbq.DeleteDeploymentToApplicationMappingByNamespaceAndName(ctx,
 		createDeplToAppMapping.DeploymentName,
 		createDeplToAppMapping.DeploymentNamespace,
 		createDeplToAppMapping.NamespaceUID); err != nil {
-		log.Error(err, "unable to delete old deployment to application mapping for name '"+
-			createDeplToAppMapping.DeploymentName+"', namespace '"+createDeplToAppMapping.DeploymentNamespace+"'")
+		log.Error(err, "Unable to delete old DeploymentToApplicationMapping, by Namespace and Name")
 		return false, err
+	} else {
+		log.Info("Deleted old DeploymentToApplicationMapping, by Namespace and Name", "resultsDeleted", resultsDeleted)
 	}
-	log.Info(fmt.Sprintf("Deleted DeploymentToApplicationMappingByNamespaceAndName with namespace: %s and name: %s", createDeplToAppMapping.DeploymentNamespace, createDeplToAppMapping.DeploymentName))
 
 	if err := dbq.CreateDeploymentToApplicationMapping(ctx, createDeplToAppMapping); err != nil {
-		log.Error(err, "unable to create deplToApp mapping", "createDeplToAppMapping", createDeplToAppMapping)
+		log.Error(err, "Unable to create DeploymentToApplicationMapping")
 		return false, err
 	}
-	log.Info("Created DeploymentToApplicationMapping: " + createDeplToAppMapping.Deploymenttoapplicationmapping_uid_id)
+	log.Info("Created DeploymentToApplicationMapping")
```

Comment:
Any specific reasons for not having `dtam GetAsLogKeyValues` here? Just checking in case we want to maintain uniformity.

Reply:
It should already be included in the log statement as part of `log = log.WithValues(createDeplToAppMapping.GetAsLogKeyValues()...)` above.